astropy/visualization/wcsaxes/formatter_locator.py

```diff
@@ -301,9 +301,11 @@ def locator(self, value_min, value_max):
 
             # In the special case where value_min is the same as value_max, we
             # don't locate any ticks. This can occur for example when taking a
-            # slice for a cube (along the dimension sliced).
+            # slice for a cube (along the dimension sliced). We return a
+            # non-zero spacing in case the caller needs to format a single
+            # coordinate, e.g. for mousover.
             if value_min == value_max:
-                return [] * self._unit, 0 * self._unit
+                return [] * self._unit, 1 * u.arcsec
```

Comment:
Why are we using arcsec here?

Reply:
This doesn't actually matter, it's just that 0 was causing issues. But any value should be fine here.